lib/dictator/unauthorized_handlers/json_api.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Dictator.UnauthorizedHandlers.JsonApi do
+  import Phoenix.Controller, only: [json: 2]
```

Comment:
Since the JSON value is always the same, can we hardcode its encoded value and set the content type manually? [It's what the implementation does](https://github.com/phoenixframework/phoenix/blob/v1.5.3/lib/phoenix/controller.ex#L283-L286)

Reply:
The reasoning for this is that we should aim to not depend on Phoenix, especially given #7

Reply:
makes sense